pkg/auth/tokens/manager.go

```diff
@@ -272,12 +279,14 @@ func (m *Manager) deriveToken(request *types.APIContext) error {
 	bytes, err := ioutil.ReadAll(r.Body)
 	if err != nil {
 		logrus.Errorf("GetToken failed with error: %v", err)
+		return httperror.NewAPIError(httperror.InvalidBodyContent, fmt.Sprintf("%s", err))
```

Comment:
dorp the error log and just return the error

Reply:
done